api/app/serializers/spree/api/v2/platform/base_serializer.rb

```diff
@@ -3,6 +3,13 @@ module Api
     module V2
       module Platform
         class BaseSerializer < ::Spree::Api::V2::BaseSerializer
+          attribute :webhook_metadata, if: proc { |_record, params|
+            params[:webhook_metadata].present? && params[:webhook_metadata] == true
```

Comment:
Could you use only `params[:webhook_metadata] == true`? If no `webhook_metadata` key in the params it'll return nil, if present then the value, which can be compared with true. It might fail if no `params` are given it seems.